opcua/common/xmlparser.py

```diff
@@ -53,7 +53,8 @@ def __init__(self):
 
 class XMLParser(object):
 
-    def __init__(self, xmlpath):
+    def __init__(self, xmlpath, server):
+        self.server = server  # POC
```

Comment:
Send necessary to implement what you proposed. If it does not break things, should be ok